examples/framework/preferences/minimalEditor/js/minEditor.js

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2015 OCAD University
+
+Licensed under the Educational Community License (ECL), Version 2.0 or the New
+BSD license. You may not use this file except in compliance with one of these
+Licenses.
+
+You may obtain a copy of the ECL 2.0 License and BSD License at
+https://github.com/fluid-project/infusion/raw/master/Infusion-LICENSE.txt
+*/
+
+/* global fluid */
+
+(function () {
+    "use strict";
+
+    /**
+     * Panel for the auto-pilot preference
+     */
+    fluid.defaults("minEditor.panels.autoPilot", {
```

Comment:
The folders for all of the examples are named based on what they're examples of. I was following that pattern. Do you think I should change it?